config/puma.rb

```diff
@@ -9,7 +9,8 @@
 
 # Specifies the `environment` that Puma will run in.
 #
-environment ENV.fetch("RAILS_ENV") { "development" }
+env = ENV.fetch("RAILS_ENV") { "development" }
```

Comment:
```
ENV.fetch('RAILS_ENV', 'development')
```

The block is just used when you don't want to automatically execute `"development"` every time it runs. For example what if that was `YAML.load(File.read(".config.yml"))[:environment]` ... you wouldn't want to execute that unless `RAILS_ENV` wasn't set.

Reply:
Done